CHANGELOG.md

```diff
@@ -6,10 +6,15 @@ Unreleased
 
 - Caching for provenance results
 - Integration tests using "real" databases and web server
-- Improved documentation and argument checking for CLI
-- GP-1862: Remove /provenance suffix from URL
```

Comment:
This line isn't moved to the changes below, is that on purpose?

Reply:
Yes, it's on purpose. I reinstated the `/provenance` suffix to avoid conflicts between the different servlets. See https://jira.oicr.on.ca/browse/GP-1862